client/rest-high-level/src/main/java/org/elasticsearch/client/eql/EqlSearchRequest.java

```diff
@@ -60,6 +61,7 @@
     static final String KEY_IMPLICIT_JOIN_KEY_FIELD = "implicit_join_key_field";
     static final String KEY_CASE_SENSITIVE = "case_sensitive";
     static final String KEY_SIZE = "size";
```

Comment:
@jrodewig A heads-up on these two parameters.
The default size of the response changed to `10` from `50` (aligned with that of the search api). 
Additionally the `fetch_size` parameters has been introduced to indicate how large the page for sequence matching is. A large page means faster results and less search calls but it does so at the expense of memory (more data needs to be returned).

Reply:
Thanks for the heads up. #59085 should already cover the changes to `size`. I'll work on documenting the `fetch_size` param.